scripts/chains/smingot_chain.zs

```diff
@@ -0,0 +1,328 @@
+	.inputs(<ore:plateDensePressCake>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 5000, 100)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+assembler.recipeBuilder()
+	.inputs(<ore:plateDensePressCakeDutched>)
+	.inputs(sensors[3])
+	.chancedOutput(<ore:plateDensePressCakeGraded>.firstItem, 7500, 300)
+	.EUt(480)
+	.duration(40)
+	.buildAndRegister();
+	
+metal_bender.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedMatterGrahamHot>)
+	.property("circuit", 0)
+	.outputs(<ore:plateDenseMatterGrahamHot>.firstItem)
+	.EUt(45)
+	.duration(80)
+	.buildAndRegister();
+	
+centrifuge.recipeBuilder()
+	.inputs(<ore:egg>)
+	.fluidOutputs(<liquid:albumen> * 100)
+	.fluidOutputs(<liquid:yolk> * 100)
+	.EUt(45)
+	.duration(60)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorPressed> * 2)
+	.outputs(<ore:plateDensePressCake>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+fluid_extractor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorDutchedPressed> * 2)
+	.outputs(<ore:plateDensePressCakeDutched>.firstItem)
+	.fluidOutputs(<liquid:cocoa_butter> * 144)
+	.EUt(120)
+	.duration(100)
+	.buildAndRegister();
+	
+furnace.addRecipe(<ore:crushedCentrifugedMatterGrahamHot>.firstItem, <ore:crushedCentrifugedMatterGraham>);
+
+var gelatins as IIngredient[] = [<ore:stickBone>, <ore:leather>, <minecraft:porkchop>, <minecraft:beef>, <minecraft:chicken>, <minecraft:rabbit>, <minecraft:mutton>];
+
+for gelatin_producer in gelatins {
+	extractor.recipeBuilder()
+		.inputs(gelatin_producer)
+		.outputs(<nuclearcraft:gelatin>)
+		.EUt(120)
+		.duration(100)
+		.buildAndRegister();
+}
+
+fluid_solidifier.recipeBuilder()
+	.fluidInputs(<liquid:fish_oil> * 400)
+	.inputs(<metaitem:shape.mold.plate>)
+	.outputs(<nuclearcraft:gelatin>)
+	.EUt(60)
+	.duration(100)
+	.buildAndRegister();
+	
+mixer.recipeBuilder()
+	.inputs(<ore:dustSugar>)
+	.inputs(<ore:dustWheat> * 3)
+	.inputs(<ore:dustSodiumBicarbonate>)
+	.fluidInputs(<liquid:butter> * 2000)
+	.fluidInputs(<liquid:milk> * 500)
+	.outputs(<ore:crushedCentrifugedMatterGraham>.firstItem * 8)
+	.EUt(80)
+	.duration(200)
+	.buildAndRegister();
+
+compressor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorRefined>)
+	.outputs(<ore:crushedCentrifugedChocolateLiquorPressed>.firstItem)
+	.EUt(360)
+	.duration(80)
+	.buildAndRegister();
+
+compressor.recipeBuilder()
+	.inputs(<ore:crushedCentrifugedChocolateLiquorDutched>)
+	.outputs(<ore:crushedCentrifugedChocolateLiquorDutchedPressed>.firstItem)
+	.EUt(360)
+	.duration(80)
+	.buildAndRegister();
+
+mixer.recipeBuilder()
+	.inputs(<ore:dustSugar>)
+	.fluidInputs(<liquid:water> * 5000)
+	.fluidInputs(<liquid:cane_syrup> * 5000)
+	.fluidOutputs(<liquid:sweetened_diluted_cane_syrup_mixture> * 10000)
+	.EUt(120)
+	.duration(260)
+	.buildAndRegister();
+
+fermenter.recipeBuilder()
```

Comment:
I feel like the fermenter is an odd choice, but I can't really think of anything better to use.